### PR TITLE
Stop talking about Pervasives in core library documentation

### PR DESCRIPTION
**Pull request by ghost**
Since `Pervasives` is now just a deprecated alias for `Stdlib`, the Manual should stop mentioning it as a real module.

### CODE REVIEW

File: Changes [hunk 1]
```diff
@@ -295,6 +295,9 @@ OCaml 4.09.0
 
 ### Manual and documentation:
 
+- #8757: Stop talking about Pervasives in core library documentation.
```
Comment (dra27) [inline Changes:298]:
```suggestion
- #8757: Rename Pervasives to Stdlib in core library documentation.
```

**Review by dra27 — COMMENTED** [review]
Thanks - I’d change the description of the change, but the change itself is of course correct.

**Comment by dra27** [comment]
I pushed a commit with a few other renames and, while there, spotted that the use of `input` as an example in the C chapter of the manual was a bit limited. This all seems good for 4.09 as well, being documentation fixes only.

**Comment by ghost** [comment]
Thanks @dra27 , that scratches my itch thoroughly.

**Comment by pmetzger** [comment]
I'm confused, did this get closed without being merged?

**Comment by ghost** [comment]
> I'm confused, did this get closed without being merged?

I thought @dra27 said he pushed a commit which included my suggestions plus other changes.  Have I misunderstood him?

**Comment by dbuenzli** [comment]
> I thought @dra27 said he pushed a commit which included my suggestions plus other changes. Have I misunderstood him?

Yes he added them on you branch (see above) but he didn't merge the PR.

**Comment by pmetzger** [comment]
Gotcha.

**Comment by dra27** [comment]
trunk: https://github.com/ocaml/ocaml/commit/466950b48f698cb2d9fbb5571a3ed203a8765044
4.08: https://github.com/ocaml/ocaml/commit/d44685bf54df51abffef558d92af7f014a88fdd4
4.09: https://github.com/ocaml/ocaml/commit/e9f72e515a540e62657e56f02ec1128ec98f074b

**Pull request closed without merging**